### ISSUE TITLE
Identify outstanding issues in the Genesis design

### ISSUE THREAD

**Issue by dnadales** [IntersectMBO/ouroboros-consensus#33]
- [ ] Produce a report on outstanding design issues and open implementation concerns. This report can be used to negotiate a contract with a vendor for implementing Genesis.
- [ ] Elicit feedback from IOG researchers on the latest Genesis design and respond to it.

**Comment by dnadales** [comment]
This is currently blocked on #393

**Comment by amesgen** [comment]
Superseded by the (concluded) Genesis SoW
